scripts/releasePack.mts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-console */
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { $ } from 'execa';
+import * as path from 'path';
+import * as fs from 'fs/promises';
+
+interface WorkspaceDefinition {
+  name: string;
+  version: string;
+  path: string;
+  private: boolean;
+}
+
+interface Manifest {
+  packages: Record<string, string>;
+}
+
+interface RunOptions {
+  packages?: string[];
+  outDir: string;
+  concurrency: number;
+}
+
+async function packWorkspace(workspace: WorkspaceDefinition, outDir: string): Promise<string> {
+  const packages: Record<string, string> = {};
+  const { stdout: zipFilePath } = await $({
+    cwd: workspace.path,
+  })`pnpm pack --pack-destination ${outDir}`;
+  packages[workspace.name] = zipFilePath;
+  return zipFilePath;
+}
+
+async function run({ packages, outDir, concurrency }: RunOptions) {
+  const allWorkspaces: WorkspaceDefinition[] = await $`pnpm m ls --depth -1 --json`.then((result) =>
```

Comment:
What does `pnpm m` do?

Reply:
> Concurrently performs some actions in all subdirectories with a `package.json` (excluding node_modules). A `pnpm-workspace.yaml` file may be used to control what directories are searched for packages.

Reply:
I couldn't find it in their docs, didn't think of running `--help`.
Anyway, I'd recommend using longhand switches and parameters in our scripts so they are more self-explanatory. Shorthands are good when interacting with the CLI manually to save a few keystrokes.